app/components/Card/index.tsx

```diff
@@ -13,13 +13,42 @@ const CardHeader = ({ children, className }: CardHeaderProps) => (
   <div className={cx(styles.header, className)}>{children}</div>
 );
 
+type CardContentProps = {
+  children: ReactNode;
+  danger?: boolean;
+  info?: boolean;
+};
+
+const CardContent = ({ children, danger, info }: CardContentProps) => {
+  let icon;
+  if (danger) {
+    icon = <Icon name="alert-circle-outline" className={styles.warningIcon} />;
+  }
+
+  if (info) {
+    icon = (
+      <Icon name="information-circle-outline" className={styles.infoIcon} />
+    );
+  }
```

Comment:
I think maybe `if (info)` should come before `if (danger)` that way if both are true, it is going to be a danger card

Reply:
That's a valid point. I'm planning on adding `warning` as well, and when I do I think I'll merge these into one prop so you'll never actually get to pass multiple severities.